### DISCUSSION TITLE
Hosting a Laravel web site

### DISCUSSION THREAD

**Discussion by LaviniaDodu** [community/community#23873] [Other Feature Feedback, Questions, & Ideas]
<p>Hello everyone, I have an issue with hosting a Laravel web site, I followed the GitHub Doc but the only thing I can see only is the ReadMe file. I’m getting crazy trying to find a solution for this, if anybody have any suggetion for me I woul be more than gratefull .<br>
This is my project <a href="https://github.com/LaviniaDodu/portafoglio" class="inline-onebox" rel="noopener nofollow ugc">GitHub - LaviniaDodu/portafoglio</a><br>
And this is what I can see online <a href="https://laviniadodu.github.io/portafoglio" rel="noopener nofollow ugc">https://laviniadodu.github.io/portafoglio</a></p>

**Comment by SubZtep** [comment, marked as answer]
<p>Hi, I doubt it’s possible, PHP runs on a server and requires a compiler, GH Pages can only host static websites including client-side JavaScript. Generate the output HTML files could work but you lose the dynamic content.</p>

**Comment by LaviniaDodu** [comment]
<p>I’m sad to hear this, anyway thank you for your answer :pray:</p>
